src/components/containers/popover/PopoverLink.js

```diff
@@ -65,7 +67,8 @@ class PopoverLink extends React.Component {
           }}
         >
           <PopoverButton
-            aria-haspopup="dialog"
+            aria-haspopup="true"
```

Comment:
If you click on that "accessibility training" link in my previous comment, you can find the example in the slides where they specifically say you can use "dialog".  I seem to recall them saying it was a new value but should work cross-browser.  However, since it's not in the official specs I shouldn't have put that there in the first place.